index.js

```diff
@@ -0,0 +1,12 @@
+const hamburgerBtn = document.querySelector('.hamburgerBtn');
+const menu = document.querySelector('.mobile-nav');
+const closeMenu = document.querySelector('#close-menu');
+hamburgerBtn.addEventListener('click', () => {
+  menu.classList.remove('hidden');
+  menu.classList.toggle('visible');
+});
+
+closeMenu.addEventListener('click', () => {
+  menu.classList.remove('visible');
+  menu.classList.toggle('hidden');
+});
```

Comment:
- [ ] @AjroudiRida and @hetrox8 great work adding the mobile menu, however,  some features are not yet implemented, and I am unable to close the menu by clicking any of the mobile menu options and to be taken to the correct part of the page depending on the option I choose. Kindly consider adding these features they are part of this milestone requirement as shown below
![image](https://github.com/AjroudiRida/Portfolio/assets/61408860/e4b6fd03-05b7-4115-8b8e-fa7e593ad9fc)